examples/server/README.md

```diff
@@ -1,8 +1,22 @@
-# llama.cpp/example/server
+# LLaMA.cpp HTTP Server
 
-This example demonstrates a simple HTTP API server and a simple web front end to interact with llama.cpp.
+Fast, lightweight, production ready pure C/C++ HTTP server based on [httplib](https://github.com/yhirose/cpp-httplib), [nlohmann::json](https://github.com/nlohmann/json) and **llama.cpp**.
```

Comment:
```suggestion
Fast, lightweight, pure C/C++ HTTP server based on [httplib](https://github.com/yhirose/cpp-httplib), [nlohmann::json](https://github.com/nlohmann/json) and **llama.cpp**.
```

Reply:
I think "Fast" should not be mention here, since we're not saying the inference is fast or the HTTP server stuff is fast (and also faster compared to what). If the inference is fast, we should say that it's fast on the llama.cpp side, not server side.